### DISCUSSION TITLE
Longhorn 1.5.4 release date?

### DISCUSSION THREAD

**Discussion by Delta1977** [longhorn/longhorn#7513] [General]
Hi, 
is there a planned release date for Longhorn 1.5.4?
[BUG] Warning events are being spammed by Longhorn - CRD https://github.com/longhorn/longhorn/issues/7290 - @m-ildefons
is annoying and Rancher Teams is also waiting for release 1.5.4 so they can make a oob update

**Comment by c3y1huang** [comment]
AFAIK, v1.5.4 is currently scheduled for release in mid-February. cc @innobead

**Reply by innobead** [reply]
Please check the milestone page on longhorn repo. It's about end of February. The first RC will be just after 1.6.0.

**Reply by c3y1huang** [reply]
@Delta1977 , ref: https://github.com/longhorn/longhorn/milestones
